client/packages/system/src/Item/DetailView/Tabs/ItemVariants/ItemVariantsTab.tsx

```diff
@@ -48,7 +48,12 @@ export const ItemVariantsTab = ({
       </AppBarButtonsPortal>
       <Box flex={1} marginX={2}>
         {itemVariants.length === 0 ? (
-          <NothingHere body={t('messages.no-item-variants')} />
+          <>
+            <NothingHere
+              body={t('messages.no-item-variants')}
+              onCreate={() => onOpen()}
+            />
+          </>
```

Comment:
Couple of minor things to change:

* the syntax `() => ` here will create a new function, which is an overhead that isn't required. The `onOpen` doesn't need parameters so can be passed directly
* The `<>...</>` isn't required. that's an empty element which you'll only need if there are multiple children and you need to provide only one

```suggestion
          <NothingHere
            body={t('messages.no-item-variants')}
            onCreate={onOpen}
          />
```

Reply:
Thanks Mark! I'll bear that in mind, and put those changes into a commit.